assignment4.Rmd

```diff
@@ -105,14 +147,144 @@ sweep(loadings, 2, colSums(loadings), "/") #sweep() computes each row as a propo
 
 #Now examine your components and try to come up with substantive descriptions of what some might represent?
 
+
 #You can generate a biplot to help you, though these can be a bit confusing. They plot the transformed data by the first two components. Therefore, the axes represent the direction of maximum variance. Then mapped onto this point cloud are the original directions of the variables, depicted as red arrows. It is supposed to provide a visualization of which variables "go together". Variables that possibly represent the same underlying construct point in the same direction.  
 
 biplot(pca)
 
 #Calculate values for each student that represent these your composite variables and then create a new correlogram showing their relationship to mean_correct.
 
+#composite values
+D4
+COR3 <- cor(cbind(D2%*%loadings, D1$mean_correct)) 
+rownames(COR3) <- c("P1","PC2","PC3","PC4","PC5","PC6","mean_correct")
+colnames(COR3) <- c("P1","PC2","PC3","PC4","PC5","PC6","mean_correct")
+corrplot(COR3, order="AOE", method="circle", tl.pos="lt", type="upper",        
+tl.col="black", tl.cex=0.6, tl.srt=45, 
+        addCoef.col="black", addCoefasPercent = TRUE,
+        sig.level=0.50, insig = "blank")
+
```

Comment:
Alternatively, `as.matrix(D2) %*% pca$rotation` or you can also calculate it manually like so: composite1 = var1 *eigenvector1 + var2*eigenvector2, etc..